src/bundle/Resources/translations/messages.en.xliff

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:jms="urn:jms:translation" version="1.2">
```

Comment:
Please add domain to translations: `matrix_fieldtype`, we should not have translations in `messages`